src/pages/Portfolio/Donations/DonationsPage.vue

```diff
@@ -272,69 +266,89 @@ import myDonationsQuery from '#src/graphql/query/portfolio/myDonations.graphql';
 import { mdiHelpCircle } from '@mdi/js';
 import KvLoadingSpinner from '#src/components/Kv/KvLoadingSpinner';
```

Comment:
There's a version of this in kv-components as well. If we move to refactor the child components at some point we could migrate this too.

Reply:
Funny thing is that I looked for a spinner and couldn't find it. Haha oops. I replaced with the shared component, it looks better anyways.

Reply:
Thanks for the callout!